app/controllers/openid_connect/authorization_controller.rb

```diff
@@ -75,6 +76,10 @@ def validate_authorize_form
       end
     end
 
+    def force_login_if_prompt_param_is_login
+      logout if @authorize_form.prompt == 'login'
```

Comment:
Did you mean `sign_out`?

Reply:
not even using this anymore.  please disregard code..

Reply:
yes it was sign_out.  however the authenticable module comes first so this is a bad strategy here

Reply:
flip flopped on sign_out.  I wasn't crazy about blowing away the session from the beginning and even stated it in our meeting before the User Story was created.  However, the alternative is passing args and conditionals where'd I'd rather not put them.